README.md

```diff
@@ -41,7 +41,23 @@ You can also preserve the whitespace in your template with the following loader
 {
   test: /\.dust$/,
   loader: 'dust-loader',
-  query: 'preserveWhitespace'
+  options: 'preserveWhitespace'
```

Comment:
Can you please also add an example of using `preserveWhitespace` unsing `options` as an object?